lib/validators/webLedgerConfiguration.js

```diff
@@ -26,10 +24,7 @@ module.exports = async ({ledgerNode, validatorInput}) => {
   if(!result.valid) {
     return result;
   }
-  // FIXME: for testnet_v2, the genesis configuration has a
```

Comment:
@dlongley ok so not 100% sure if removing this FIXME is valid. We are using a `CapabilityInvocation` purpose now, but do any further changes need to be made? One idea I had is that we might want to search for the original ledgerConfig block, and then set the expected controller to the controller of the LedgerConfig block.